drush/Commands/cgov/CgovUserCommands.php

```diff
@@ -90,6 +91,42 @@ public function blockAdminUser() {
 
   }
 
+  /**
+   * UnBlock Admin User.
+   *
+   * @command cgov:unblock-admin-user
+   * @aliases unblock-admin-user, ublkadm
+   * @usage drush cgov:unblock-admin-user
+   *   UnBlock the admin user (user 1).
+   * @bootstrap full
+   */
+  public function unblockAdminUser() {
+    /** @var \Drupal\user\Entity\User $account */
+    if ($account = User::load(1)) {
+      $account->activate();
+      $account->save();
+      if (\Drupal::moduleHandler()->moduleExists('samlauth')) {
```

Comment:
You'll actually want to check to see if `externalauth` exists, not `samlauth`. External Authentication is the module that controls that table.